lib/mongify/translation/sync.rb

```diff
@@ -63,7 +63,8 @@ def sync_data
           max_updated_at, max_updated_at_id = Time.new(1970), nil
           rows.each do |row|
             row_hash = t.translate(row)
-            updated_at = Time.parse(row['updated_at'])
+            updated_at = row['updated_at']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.